schema/musicxml.xsd

```diff
@@ -4630,25 +4630,21 @@ The repeater attribute has been deprecated in MusicXML 3.0. Formerly used for tr
 
 	<xs:complexType name="bend">
 		<xs:annotation>
-			<xs:documentation>The bend type is used in guitar notation and tablature. The bend-alter element indicates the number of steps in the bend, similar to the alter element. As with the alter element, numbers like 0.5 can be used to indicate microtones. Negative numbers indicate pre-bends or releases; the pre-bend and release elements are used to distinguish what is intended. The shape attribute distinguishes between the angled bend symbols commonly used in standard notation and the curved bend symbols commonly used in both tablature and standard notation.</xs:documentation>
+			<xs:documentation>The bend type is used in guitar notation and tablature. Notes with a bend and release will have two bend elements, with the release following the bend. The shape attribute distinguishes between the angled bend symbols commonly used in standard notation and the curved bend symbols commonly used in both tablature and standard notation.</xs:documentation>

```

Comment:
> Notes with a bend and release will have two bend elements, with the release following the bend.

The term "bend" in this sentence feels a bit overloaded and might be confusing; it's being used both to mean the concept and the MusicXML element. What about rewording to something like this:

"If a single note contains a bend and release, use two bend elements, one after another -- one element to represent the bend and another element to represent the release."